packages/controllers/src/utils.ts

```diff
@@ -109,11 +108,7 @@ export async function withTimeout<PromisValue = void>(
  *     assertExhaustive(snapPrefix);
  * }
  * ```
- * @param _ - The object on which the switch is being operated
- */
-/* istanbul ignore next */
-/**
- * @param _
+ * @param _ - The object on which the switch is being operated.
```

Comment:
What's this for? And can you name the parameter, like `_object`?

Reply:
Utility function for this: https://github.com/MetaMask/snaps-skunkworks/blob/127091c8a9cc07a66ad3c6f3d1f69f8c2f6bc510/packages/controllers/src/snaps/SnapController.ts#L1627-L1628

Can name it 👍